tests/integration/plugin_test.go

```diff
@@ -0,0 +1,25 @@
+//+build integration
+//+build !windows
+
+package integration
+
+import (
+  . "gopkg.in/check.v1"
```

Comment:
This import doesn't have consistent spacing.

Reply:
fixed